.gitmodules

```diff
@@ -1,16 +1,16 @@
 [submodule "source/RailgunNet"]
 	path = source/RailgunNet
-	url = git@github.com:araex/RailgunNet.git
-	branch = netstandard
+	url = git@github.com:evorios/RailgunNet.git
```

Comment:
What is the reason for changing this?

Reply:
Because  https://github.com/araex/RailgunNet/pull/1 needed to be approved first. It is merged now. 

@evorios I also merged `netstandard` back to master. Can you please change the url back to what it was and the branch to `master`?